packages/legacy/core/App/components/views/FauxNavigationBar.tsx

```diff
@@ -9,7 +9,7 @@ import { GenericFn } from '../../types/fn'
 import { testIdWithKey } from '../../utils/testable'
 
 const defaultIconSize = 24
-const defaultStatusAndNavbarHeight = 44
+const defaultStatusAndNavbarHeight = Platform.OS === 'ios' ? 44 : undefined
```

Comment:
Same thing here, just curious what the reason is for the difference in platforms

Reply:
The same gap appears here. 
![unnamed](https://user-images.githubusercontent.com/112953503/234386959-c9f64bd9-e2fd-4c26-9667-a693de6f54a0.png)